src/app/messages.ts

```diff
@@ -38,12 +60,39 @@ export const errMessages = {
     }
   },
   Login: {
-    
+    "login": "Oops... Invalid email or password, please type it again."
+  },
+  PasswordValidation: {
+    mismatch: {
+      "mismatch": "Password mismatch ..."
+    },
+    minlength: {
+      "minlength": "Password minimum length is 8 characters ..."
+    }
   },
   Registration: {
-
+    mismatch: {
+      "match": "The passwords entered do not match."
+    },
+    error: {
+      "error": "Sorry, something went wrong. Please contact " // after the "contact", the content will be a variable, please check with Registration/model.ts file to view detail
```

Comment:
If this is the generic error message just for registrations
"Sorry, something has gone wrong in processing your registration. Please try again or contact"

Otherwise in what situation is this error message called?